src/ifcwrap-csharp/IfcOpenShell.Net/IfcOpenShell.Net/IfcFileExtensions.cs

```diff
@@ -0,0 +1,55 @@
+using System;
+
+namespace IfcOpenShell
+{
+    public static class IfcFileExtensions
+    {
+
+        /// <summary>
+        /// Returns a unit scale factor to convert to and from IFC project units and SI units.
+        /// </summary>
+        /// <returns>The scale factor</returns>
+        public static double CalculateUnitScale(this IfcFile file, string unitType = "LENGTHUNIT")
+        {
+            var unitAssignment = file.ByType("IfcUnitAssignment");
+            if (unitAssignment == null || unitAssignment.Count == 0)
+            {
+                return 1.0;
```

Comment:
I assume return 1 is fine because every unit should be equal to SI units? Or is this just a fallback?